roles/confluent.kafka_connect/defaults/main.yml

```diff
@@ -1,13 +1,12 @@
 ---
 # More kafka_connect variables are defined in the confluent.variables role
 
-### Boolean to enable cp-ansible's Custom Log4j Configuration
+### Boolean to reconfigure Connect's logging with RollingFileAppender and log cleanup
```

Comment:
Boolean to reconfigure Kafka Connect's logging with the RollingFileAppender and log cleanup functionality.